### PR TITLE
docs: added upgrade notes DOC-773

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds an update section to the documentation and updates the breaking change section.

## Review Changes

💻 [ Preview URL](https://deploy-preview-1307--docs-spectrocloud.netlify.app/)

🎫 [DOC-773](https://spectrocloud.atlassian.net/browse/DOC-773)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1307#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md
Comment (ritawatson) [inline content/docs/00-release-notes.md:29]:
```suggestion
- Self-hosted Palette for Kubernetes now installs Palette Ingress resources in a namespace that Palette manages. Prior versions of Palette installed the internal components of ingress resources in the default namespace. Review the [Upgrade Notes](/enterprise-version/upgrade#palette3.4) to learn more about this change and how to upgrade.
```

File: content/docs/00-release-notes.md
Comment (ritawatson) [inline content/docs/00-release-notes.md:244]:
```suggestion
- Kubernetes versions before 1.24 are no longer supported for host clusters targeting Azure Kubernetes Service (AKS). This deprecation is due to Azure's Kubernetes support policy. You can learn more about Azure-supported Kubernetes versions [here](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli).
```

File: content/docs/12-enterprise-version.md
Comment (ritawatson) [inline content/docs/12-enterprise-version.md:45]:
```suggestion
Palette can be installed in a VMware environment without internet access, known as an air gap installation, requiring pre-download of platform manifests, required platform packages, container images for core components, third-party dependencies, and Palette Packs, all sourced from a private rather than the default public Palette repository.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:267]: @karl-cardenas-coding Don't we typically use the long form of the command options?

Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:267]:
```suggestion
  helm install palette /path/to/chart.tgz -file /path/to/values.yaml
```

File: content/docs/12-enterprise-version/10-upgrade.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/10-upgrade.md:4]:
```suggestion
metaDescription: "Spectro Cloud upgrades notes for specific Palette versions."
```

File: content/docs/12-enterprise-version/10-upgrade.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/10-upgrade.md:19]:
```suggestion
Prior versions of Palette installed internal Palette components' ingress resources in the default namespace. The new version of the Helm Chart ensures all Palette required ingress resources are installed in the correct namespace. Self-hosted Palette instances deployed to Kubernetes and upgrading from Palette versions 3.3.X or older must complete the following action.
```

File: content/docs/14-troubleshooting/15-palette-upgrade.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/15-palette-upgrade.md:22]: Incomplete sentence. Did you mean to include the sentence "Use the following steps..." here? If so, you can take the suggestion for line 22 and delete line 30.

Comment (ritawatson) [inline content/docs/14-troubleshooting/15-palette-upgrade.md:22]:
```suggestion
If you receive the following error message when attempting to upgrade to Palette versions greater than Palette 3.4.X in a Kubernetes environment, use the debugging steps to address the issue.
```

File: content/docs/14-troubleshooting/15-palette-upgrade.md
Comment (ritawatson) [inline content/docs/14-troubleshooting/15-palette-upgrade.md:30]: Delete line.

**Review by ritawatson — APPROVED** [review]
LGTM with a couple of suggestions. Well done.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1307#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1307#pullrequestreview-1437294169" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/14-troubleshooting/15-palette-upgrade.md
Comment (karl-cardenas-coding) [inline content/docs/14-troubleshooting/15-palette-upgrade.md:30]:
```suggestion
```

**Pull request merged**
